### ISSUE TITLE
Since upgrade to 1.1.0: Command works without parameter, but not with parameter anymore

### ISSUE THREAD

**Issue by finn-matti** [laminas/laminas-cli#74]
### BC Break Report

|    Q        |   A
|------------ | ------
| Version     | 1.1.0

#### Summary

(I'm not sure what BC stand for, but the template description made it seem that this is the right one for my issue.)

Since the update today my command has stopped working. When executing the command without the parameter I get the expected message:

```
./vendor/bin/laminas app:publish

Not enough arguments (missing: "message"). 
```

If I add the message I get this error:

```
./vendor/bin/laminas app:publish foo

There are no commands defined in the "app" namespace.
```

#### Previous behavior

The command works as expected.

#### Current behavior

The command doesn't work.

#### How to reproduce

I don't have a unit test because I still try to get more comfortable with laminas-cli. Here's the `module.config.php`:

```
<?php

/**
 * @see       https://github.com/laminas/laminas-mvc-skeleton for the canonical source repository
 *
 * @copyright https://github.com/laminas/laminas-mvc-skeleton/blob/master/COPYRIGHT.md
 * @license   https://github.com/laminas/laminas-mvc-skeleton/blob/master/LICENSE.md New BSD License
 */

declare(strict_types=1);

namespace Worker;

use Laminas\ServiceManager\Factory\InvokableFactory;
use Worker\Command\HelloCommand;
use Worker\Command\PublishCommand;

return [
    'laminas-cli'     => [
        'commands' => [
            'app:hello'   => HelloCommand::class,
            'app:publish' => PublishCommand::class,
        ],
    ],
    'service_manager' => [
        'factories' => [
            HelloCommand::class    => InvokableFactory::class,
            PublishCommand::class  => InvokableFactory::class,
        ],
    ],
];
```

**Comment by boesing** [comment]
Thanks for reporting this. Gonna try to create a test for that later on.

**Comment by boesing** [comment]
@finn-matti might want to try #75?

**Comment by finn-matti** [comment]
Thanks @boesing! I'll try to do so later today.

**Comment by boesing** [comment]
@finn-matti maybe give #76 the first try as I guess due to simplicity, that will be the final solution.

**Comment by finn-matti** [comment]
@boesing FYI: Both #75 and #76 worked for me.

**Comment by weierophinney** [comment]
Fixed with #76; release of 1.1.1 coming shortly.
